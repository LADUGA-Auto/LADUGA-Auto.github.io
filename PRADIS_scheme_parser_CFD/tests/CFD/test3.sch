<Qucs Schematic 4.4.0>
<Properties>
  <View=0,0,800,800,1,0,0>
  <Grid=10,10,1>
  <DataSet=test3.dat>
  <DataDisplay=test3.dpl>
  <OpenDisplay=1>
  <showFrame=0>
  <FrameText0=\x041D\x0430\x0437\x0432\x0430\x043D\x0438\x0435>
  <FrameText1=\x0427\x0435\x0440\x0442\x0438\x043B:>
  <FrameText2=\x0414\x0430\x0442\x0430:>
  <FrameText3=\x0412\x0435\x0440\x0441\x0438\x044F:>
</Properties>
<Symbol>
</Symbol>
<Components>
  <CoSimulation.Port Port1 1 160 50 -41 29 0 0 "P1" 0 "Fluid" 0>
  <CoSimulation.Port Port2 1 160 120 -41 29 0 0 "" 0 "ThermalFluid" 0>
  <CoSimulation.Port Port3 1 160 190 -41 29 0 0 "P3" 0 "Thermal" 0>
  <CoSimulation.Port Port4 1 160 260 -41 29 0 0 "P4" 0 "XYZ" 0>
  <CoSimulation.Port Port5 1 400 50 -41 29 0 0 "P5" 0 "Point" 0>
  <CoSimulation.Port Port6 1 400 120 -41 29 0 0 "P6" 0 "Point2d" 0>
  <CoSimulation.Port Port7 1 400 190 -41 29 0 0 "P7" 0 "DOF1" 0>
  <CoSimulation.Port Port8 1 400 260 -41 29 0 0 "P8" 0 "Electrical" 0>
  <CoSimulation.Task Task1 1 190 470 -41 29 0 0 "resulfile" 0 "resultDesc" 0>
  <CoSimulation.SolidDomain SolidDomain1 1 280 140 -71 29 0 0 "stone" 0 "10.0" 0 "2.6" 0 "45.1" 0>
  <CoSimulation.Solver Solver1 1 280 330 -51 29 0 0 "60.6" 0 "10" 0 "0.5" 0 "" 0>
</Components>
<Wires>
</Wires>
<Diagrams>
</Diagrams>
<Paintings>
</Paintings>
