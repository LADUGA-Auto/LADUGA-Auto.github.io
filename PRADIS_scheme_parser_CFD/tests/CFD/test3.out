[ Module ]      |      [ Name ]       |      [ Active ]       |      [ Properties ]
[ CoSimulation.Port ] | [ Port1 ] | [ true ]  |  [ Name : P1 : string ]    [ Interface : Fluid : string ]    

[ CoSimulation.Port ] | [ Port2 ] | [ true ]  |  [ Name :  : string ]    [ Interface : ThermalFluid : string ]    

[ CoSimulation.Port ] | [ Port3 ] | [ true ]  |  [ Name : P3 : string ]    [ Interface : Thermal : string ]    

[ CoSimulation.Port ] | [ Port4 ] | [ true ]  |  [ Name : P4 : string ]    [ Interface : XYZ : string ]    

[ CoSimulation.Port ] | [ Port5 ] | [ true ]  |  [ Name : P5 : string ]    [ Interface : Point : string ]    

[ CoSimulation.Port ] | [ Port6 ] | [ true ]  |  [ Name : P6 : string ]    [ Interface : Point2d : string ]    

[ CoSimulation.Port ] | [ Port7 ] | [ true ]  |  [ Name : P7 : string ]    [ Interface : DOF1 : string ]    

[ CoSimulation.Port ] | [ Port8 ] | [ true ]  |  [ Name : P8 : string ]    [ Interface : Electrical : string ]    

[ CoSimulation.Task ] | [ Task1 ] | [ true ]  |  [ File : resulfile : string ]    [ Description : resultDesc : string ]    

[ CoSimulation.SolidDomain ] | [ SolidDomain1 ] | [ true ]  |  [ Solid : stone : string ]    [ Velocity : 10 : double ]    [ Temperature : 2.6 : double ]    [ RotationalVelocity : 45.1 : double ]    

[ CoSimulation.Solver ] | [ Solver1 ] | [ true ]  |  [ EndTime : 60.6 : double ]    [ MaximalIterationCount : 10 : integer ]    [ Tolerance : 0.5 : double ]    [ Flag.GetJacobian :  : string ]    

